src/app/execution/txRunner.js

```diff
@@ -42,15 +60,51 @@ TxRunner.prototype.execute = function (args, callback) {
       data: data,
       value: value
     }
+
     if (args.useCall) {
+      function execute () {
+        executionContext.web3().eth.call(tx, function (error, result) {
+          callback(error, {
+            result: result,
+            transactionHash: result.transactionHash
+          })
+        })
+      }
       tx.gas = gasLimit
-      executionContext.web3().eth.call(tx, function (error, result) {
-        callback(error, {
-          result: result,
-          transactionHash: result.transactionHash
+      if (!self.config.get('doNotShowAgain')) {
```

Comment:
in case of a `call` ( `useCall` is `true` ), the modal should not be displayed: we are not going to create a tx in that case